src/software/jetson_nano/services/motor.cpp

```diff
@@ -386,21 +386,41 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
     motor_status.mutable_angular_velocity()->set_radians_per_second(
         current_euclidean_velocity[2]);
 
-    WheelSpace_t target_wheel_velocities = {0.0, 0.0, 0.0, 0.0};
     int target_dribbler_rpm =
         motor.drive_control_case() ==
                 TbotsProto::MotorControl::DriveControlCase::DRIVE_CONTROL_NOT_SET
             ? 0
             : motor.dribbler_speed_rpm();
-    static int ramp_rpm = 0;
+    ramp_rpm = 0;
```

Comment:
This should be move to the constructor now (or the header file), or else the velocity is reset to 0 everytime

Reply:
oh i see, so the line `static int ramp_rpm = 0` would only initialise it once to 0 since it is a static variable?

Reply:
yep exactly. That line will only run once